kubernetes/loculus/templates/loculus-backend.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           ports:
             - containerPort: 8079
           args:
+            - "--loculus.enable-seqset-citations={{ $.Values.seqSets.enabled }}"
```

Comment:
(not blocking) I find this name "enable-seqset-citations" weird - its seqsets themselves that are affected not merely "seqset citations" (which I would think of as about citations of seqsets, tracking of which is a feature yet to come

Reply:
I'll rename to `enable-seqsets'